Neuron/Sections/Transaction/TransactionParamBuilder.swift

```diff
@@ -99,6 +142,23 @@ class TransactionParamBuilder: NSObject {
         }
     }
 
+    private func fetchTokenPrice(token: TokenModel) {
```

Comment:
Let's move token price logic out of param builder (as token price should not affect tx building).  (this rule should apply to estimated gas as well)

But let's do it in a future PR.